automatic/sonarlint-vs2019/legal/VERIFICATION.txt

```diff
@@ -0,0 +1,19 @@
+VERIFICATION
+ 
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+ 
+Package can be verified like this:
+ 
+1. Download: 
+   x64: https://github.com/SonarSource/sonarlint-visualstudio/releases/download/v4.16.0.14364/SonarLint.VSIX-4.16.0.14364-2019.vsix
+  
+2. You can use one of the following methods to obtain the SHA256 checksum:
+   - Use powershell function 'Get-FileHash'
+   - Use Chocolatey utility 'checksum.exe'
+ 
+   checksum type: sha256
+   checksum64: CD14E547D9E3253338F2C5628BC4363B767056C994BAA616B9845B7A60529CC8
+ 
+File 'LICENSE.txt' is obtained from:
+   https://marketplace.visualstudio.com/items/SonarSource.SonarLintforVisualStudio2019/license
```

Comment:
Since binary is downloaded from GitHub and not Marketplace I would take license file from https://github.com/SonarSource/sonarlint-visualstudio/blob/master/LICENSE which in this case applies to the binary.